Documentation/cmdref/cilium-agent.md

```diff
@@ -113,6 +113,7 @@ cilium-agent [flags]
       --k8s-watcher-queue-size uint                           Queue size used to serialize each k8s event type (default 1024)
       --keep-bpf-templates                                    Do not restore BPF template files from binary
       --keep-config                                           When restoring state, keeps containers' configuration in place
+      --kube-proxy-replacement string                         Detect and enable individual features for kube-proxy replacement ("strict"), or enable only selected features and panic if dependencies between features are broken ("strict"), or enable only selected features regardless dependencies ("partial")  (default "strict")
```

Comment:
Shall we add "disabled" as well? It's not so straightforward to disable it ("strict" + each feature has to be disabled).

Reply:
yeah, a disable here would be more explicit.